resources/lib/vrtplayer.py

```diff
@@ -34,49 +34,49 @@ def show_main_menu(self):
-                      info_dict=dict(plot=localize(30017))),
+                      art_dict=dict(thumb='DefaultTags.png', fanart='DefaultTags.png'),
+                      info_dict=dict(title=localize(30016), plot=localize(30017))),
             TitleItem(label=localize(30018),  # Live TV
                       path=url_for('livetv'),
-                      art_dict=dict(thumb='DefaultTVShows.png'),
-                      info_dict=dict(plot=localize(30019))),
+                      art_dict=dict(thumb='DefaultTVShows.png', fanart='DefaultTVShows.png'),
+                      info_dict=dict(title=localize(30018), plot=localize(30019))),
             TitleItem(label=localize(30020),  # Recent items
                       path=url_for('recent'),
-                      art_dict=dict(thumb='DefaultRecentlyAddedEpisodes.png'),
-                      info_dict=dict(plot=localize(30021))),
+                      art_dict=dict(thumb='DefaultRecentlyAddedEpisodes.png', fanart='DefaultRecentlyAddedEpisodes.png'),
+                      info_dict=dict(title=localize(30020), plot=localize(30021))),
             TitleItem(label=localize(30022),  # Soon offline
                       path=url_for('offline'),
-                      art_dict=dict(thumb='DefaultYear.png'),
-                      info_dict=dict(plot=localize(30023))),
+                      art_dict=dict(thumb='DefaultYear.png', fanart='DefaultYear.png'),
+                      info_dict=dict(title=localize(30022), plot=localize(30023))),
             TitleItem(label=localize(30024),  # Featured content
                       path=url_for('featured'),
-                      art_dict=dict(thumb='DefaultCountry.png'),
-                      info_dict=dict(plot=localize(30025))),
+                      art_dict=dict(thumb='DefaultCountry.png', fanart='DefaultCountry.png'),
+                      info_dict=dict(title=localize(30024), plot=localize(30025))),
             TitleItem(label=localize(30026),  # TV guide
                       path=url_for('tvguide'),
-                      art_dict=dict(thumb='DefaultAddonTvInfo.png'),
-                      info_dict=dict(plot=localize(30027))),
+                      art_dict=dict(thumb='DefaultAddonTvInfo.png', fanart='DefaultAddonTvInfo.png'),
+                      info_dict=dict(title=localize(30026), plot=localize(30027))),
             TitleItem(label=localize(30028),  # Search
                       path=url_for('search'),
-                      art_dict=dict(thumb='DefaultAddonsSearch.png'),
-                      info_dict=dict(plot=localize(30029))),
+                      art_dict=dict(thumb='DefaultAddonsSearch.png', fanart='DefaultAddonsSearch.png'),
+                      info_dict=dict(title=localize(30028), plot=localize(30029))),
         ])
-        show_listing(main_items, cache=False)  # No category
+        show_listing(main_items, cache=False, content='tvshows')  # No category
```

Comment:
This will show "TV shows / VRT NU" instead of "Videos / VRT NU" in the breadcrumbs of the main menu

Reply:
I changed it to 'videos'.